tasks/webfont.js

```diff
@@ -48,6 +40,24 @@ module.exports = function(grunt) {
 			}
 		};
 
+		var allDone = this.async();
+		var params = this.data;
+		var options = this.options();
+		var md5 = crypto.createHash('md5');
+
+		/*
+		 * Check for `src` param on target config
+		 */
+		this.requiresConfig([this.name, this.target, 'src'].join('.'));
+
+		/*
+		 * Check for `dest` param on either target config or global options object
+		 */
+		if (_.isUndefined(params.dest) && _.isUndefined(options.dest)) {
+			logger.error(chalk.yellow('Warning: Required property ' + [this.name, this.target, 'dest'].join('')
```

Comment:
Colorization and “warning” should be unnecessary here because in the end it will be printed by grunt.warn.